### DISCUSSION TITLE
Dry evaporator (indoor unit) after cooling with Fan-Mode

### DISCUSSION THREAD

**Discussion by rug1024** [revk/xESP32-Faikout#468] [Ideas]
When switching the unit off after cooling the evaporator (indoor unit) is usually wet, an ideal breeding ground for mold and bacteria, causing smelly air output the next time the unit is turned on again.

My idea: every time when status "COOL Power ON" is changed to "COOL Power OFF" the unit is automatically set to "FAN Power ON" for 15-30 Minutes to dry the indoor unit.

Maybe an option can be added for this? Of course this can also be self-programmed in Home Assistant or ioBroker, but not every user has these possibilities.

**Comment by revk** [comment]
Interesting idea, but may have to wait a bit.
